vendor/github.com/elastic/go-ucfg/CHANGELOG.md

```diff
@@ -120,6 +120,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unreleased]: https://github.com/elastic/go-ucfg/compare/v0.3.7...HEAD
 [0.3.7]: https://github.com/elastic/go-ucfg/compare/v0.3.6...v0.3.7
+[Unreleased]: https://github.com/elastic/go-ucfg/compare/v0.3.6...HEAD
```

Comment:
@urso Now sure how this sneaked in. Can you clean that up in https://github.com/elastic/beats/pull/3045